Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+
+// MARK: - Internal Methods
+//
+extension SPAppDelegate {
+
+    /// Returns the actual Selected Tag Name **Excluding** navigation tags, such as Trash or Untagged Notes.
+    ///
+    /// TODO: This should be gone... **the second** the AppDelegate is Swift-y. We should simply keep a `NoteListFilter` instance.
+    ///
+    @objc
+    var filteredTagName: String? {
```

Comment:
As a temp fix that will go away, I think this is fine.  If it was gonna last a while I wonder if it would be better to compose this as a method that accepted the a tag (the current tag in this case) and then performed the filter operation on it. Written that way it would be easier to have a unit test for the behavior.   And speaking of unit tests.... is it worth writing one for this?

Reply:
**Definitely** going away soon, I've already (nearly) Swifted all the new things. Okay, not everything, but a big majority 😄 

I actually tried that approach (filtering method). Thing is, I ended up with code duplication, because the enum `NoteListFilter` has associated values, and cannot be bridged to ObjC. 

Thank you for bringing this up Eric!! This (really simple PR) had two or three iterations, until I just picked the shortest one 🔥